code/src/UI/Views/NewProject/MainPage.xaml

```diff
@@ -176,23 +176,13 @@
             </Grid.ColumnDefinitions>
             <TextBlock
                 Grid.Column="0"
-                Text="{x:Static strings:StringRes.TermsAndConditionsText}">
-                <TextBlock.Style>
-                    <Style TargetType="TextBlock" BasedOn="{StaticResource WtsTextBlockTermsAndConditions}">
-                        <Setter Property="Visibility" Value="Visible" />
-                        <Style.Triggers>
-                            <DataTrigger Binding="{Binding ProjectType.Items.Count}" Value="0">
-                                <Setter Property="Visibility" Value="Collapsed" />
-                            </DataTrigger>
-                        </Style.Triggers>
-                    </Style>
-                </TextBlock.Style>
-            </TextBlock>
```

Comment:
This style trigger should not be removed, it's used to hide terms and conditions when no content is available

Reply:
removal was accidental - probably as part of a large merge